### PR TITLE
Reload sentry-kubernetes deployment after configuration change

### PR DESCRIPTION
**Pull request by gianrubio**

### CODE REVIEW

**Comment by timstoop** [comment]
/lgtm
I'm not entirely sure if you're adding a feature or not, so whether this would warrant a minor version increase instead of patch level. Not against just increase the patch level for this, though.

File: incubator/sentry-kubernetes/templates/deployment.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ spec:
       app: {{ template "sentry-kubernetes.name" . }}
   template:
     metadata:
+      annotations:
+        checksum/secrets: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```
Comment (unguiculus) [inline incubator/sentry-kubernetes/templates/deployment.yaml]: The `$` symbol is unnecessary in this case.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Pull request merged**